tests/cypress/integration/issue_1785_propagation_latest_frame.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Check propagation work from the latest frame', () => {
+
+    const issueId = '1785'
+    const labelName = `Issue ${issueId}`
+    const taskName = `New annotation task for ${labelName}`
+    const attrName = `Attr for ${labelName}`
+    const textDefaultValue = 'Some default value for type Text'
+    const images = [`image_${issueId}_1.png`,
+                    `image_${issueId}_2.png`,
+                    `image_${issueId}_3.png`]
+    const width = 800
+    const height = 800
+    const posX = 10
+    const posY = 10
+    const color = 'gray'
+    const archiveName = `images_issue_${issueId}.zip`
+    const archivePath = `cypress/fixtures/${archiveName}`
+    const imagesFolder = `cypress/fixtures/image_issue_${issueId}`
+    const directoryToArchive = imagesFolder
+
+    before(() => {
+        cy.visit('auth/login')
+        cy.login()
+        for (let img of images) {
+            cy.imageGenerator(imagesFolder, img, width, height, color, posX, posY, labelName)
+        }
+        cy.wait(2000)
+        cy.createZipArchive(directoryToArchive, archivePath)
+        cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, archiveName)
+        cy.openTaskJob(taskName)
+    })
+
+    describe(`Testing issue "${issueId}"`, () => {
+        it('Go to the last frame', () => {
+            cy.get('.cvat-player-buttons')
+            .find(':nth-child(7)')
+            .click()
+            cy.get('.ant-input-number-input')
```

Comment:
We have our defined class ``cvat-player-frame-selector``. Can we use it to achieve the element?

Reply:
An element with this class has no numerical value. Together with him I am looking for "input[role="spinbutton"]" and checking the value.

Reply:
It is okay. Just ``Antd`` input number element in DOM transforms to html div element with some internal div and input elements.